gravitee-apim-portal-webui-next/src/app/app.routes.ts

```diff
@@ -245,6 +247,24 @@ export const routes: Routes = [
       },
     ],
   },
+  {
+    path: 'documentation',
+    canActivate: [redirectGuard, authGuard],
+    children: [
+      {
+        path: '',
+        data: { breadcrumb: { alias: 'pageName' } },
+        resolve: { data: documentationResolver },
```

Comment:
You can simplify this a bit by putting:
```suggestion
        resolve: { navItem: documentationResolver },
```
And then in your documentationResolver, have it return a `PortalNavigationItem` directly instead of an object. The resolved object will be automatically added into the "data" attribute of the router.

Reply:
Done